runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -10814,11 +10814,15 @@ static TR::Register* inlineHasNegatives(TR::Node* node, TR::CodeGenerator* cg)
    // Examine the chunk register now that all of the residual bytes have been ORed into it
    generateLabelInstruction(TR::InstOpCode::label, node, residualTestLabel, cg);
 
+   // Zero out the result register
+   generateRegRegInstruction(TR::InstOpCode::XOR8RegReg, node, resultReg, resultReg, cg);
```

Comment:
These opcodes wont work on 32-bit. Use `TR::InstOpCode::XORRegReg(is64Bit);` instead. If you really need 64-bits, then disable these intrinsics on 32-bit for now. Check the other instructions for similar issues.